backend/server/server.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const DBimports = require('../database/database');
+
+const app = express();
+
+const PORT = process.env.PORT || 3000;
```

Comment:
you need to require dotenv....

if you test console.log(process.env.PORT) its not going to work...and will always default to 3000..as process.env.PORT will always be undefined